objectherkenning_openbare_ruimte/create_aml_environment/oor-environment.Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.8-cudnn8-ubuntu22.04 AS base-image
+FROM mcr.microsoft.com/azureml/openmpi4.1.0-cuda11.8-cudnn8-ubuntu22.04:20240614.v1 AS base-image
 
 # Upgrade and install system libraries
 RUN apt-get -y update \
-    && ACCEPT_EULA=Y apt-get upgrade -qq \
```

Comment:
So we *did* have this line in there but I accidentally removed it in my earlier commit. That means this was also not the magic trick that fixed everything, because we had the issues before.. :D